src/cascadia/TerminalSettingsEditor/Profiles.h

```diff
@@ -11,17 +11,24 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 {
     struct Profiles : ProfilesT<Profiles>
     {
-    public:
+   
         Profiles();
         Profiles(winrt::Microsoft::Terminal::Settings::Model::Profile profile);
 
+        void ColorSchemeSelectionChanged(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::Controls::SelectionChangedEventArgs const& args);
+        Windows::Foundation::Collections::IObservableVector<winrt::hstring> ColorSchemeList();
+
         fire_and_forget BackgroundImage_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::RoutedEventArgs const& e);
         fire_and_forget Commandline_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::RoutedEventArgs const& e);
         // TODO GH#1564: Settings UI
         // This crashes on click, for some reason
         //fire_and_forget StartingDirectory_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::RoutedEventArgs const& e);
 
         GETSET_PROPERTY(winrt::Microsoft::Terminal::Settings::Model::Profile, Profile);
+
+    private:
+        Windows::Foundation::Collections::IObservableVector<winrt::hstring> _ColorSchemeList{ nullptr };
```

Comment:
@leonMSFT + @DHowett We're creating a separate list of color schemes here. But it updates when the page loads so we should be fine. We looked into making the `ColorSchemes`'s list static, but that proved to be a bit of a pain (we'd have to update the list each time one of the pages loads, and ownership seems funky). I'm ok with this for now, but I wanted to bring it to your attention.

Reply:
Why would you use a static list? Why can't you bind the ItemsSource to _the actual list of real colorschemes_ like you did for Profiles?

Reply:
Because it's a map of color schemes, not a list :/. Or does XAML have a way to convert that smartly?